srcs/vm/utils_2.c

```diff
@@ -177,3 +177,24 @@ int		is_registry(int arg)
 		return (true);
 	return (false);
 }
+
+void		set_champ_zero(t_arena *arena, int i)
+{
+	arena->champions[i].id = 0;
```

Comment:
Sure this is needed? De hele arena struct wordt al bzero gedaan, de champions zijn static en worden dus ook op null gezet.

Reply:
Yes after I relocated the struct, otherwise you have a duplicate.